x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -189,6 +189,12 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
         const { renderApp, services, store } = await mountDependencies();
         const { ManagementSettings } = await this.lazyAssistantSettingsManagement();
 
+        services.chrome.docTitle.change(
+          i18n.translate('xpack.securitySolution.securityAiAssistantManagement.app.titleBar', {
+            defaultMessage: 'AI Assistant for Security Settings',
+          })
+        );
+
```

Comment:
Can we move this to the `ManagementSettings` component? We should minimize the amount of page-specific code we put in the main _plugin.tsx_ file.